preconditions-assertj/build.gradle

```diff
@@ -7,3 +7,7 @@ dependencies {
 
     testImplementation 'junit:junit'
 }
+
+versionsLock {
+    testProject()
+}
```

Comment:
Debatable whether this is really a `testProject()` - it does actually publish a jar that users are intended to consume, it just happens that they're intended to consume it within _their_ tests